src/containers/Cluster/Cluster.tsx

```diff
@@ -101,67 +102,127 @@ function Cluster({
+                                .pathname
+                        }
+                    >
+                        <ClusterInfo
+                            cluster={cluster}
+                            groupsStats={groupsStats}
+                            versionsValues={versionsValues}
+                            loading={infoLoading}
+                            error={clusterError}
+                            additionalClusterProps={additionalClusterProps}
+                        />
+                    </Route>
+                    <Route
+                        path={
+                            getLocationObjectFromHref(getClusterPath(clusterTabsIds.tenants))
+                                .pathname
+                        }
+                    >
+                        <Tenants additionalTenantsProps={additionalTenantsProps} />
+                    </Route>
+                    <Route
+                        path={
+                            getLocationObjectFromHref(getClusterPath(clusterTabsIds.nodes)).pathname
+                        }
+                    >
+                        <NodesWrapper
+                            parentContainer={container.current}
+                            additionalNodesProps={additionalNodesProps}
+                        />
+                    </Route>
+                    <Route
+                        path={
+                            getLocationObjectFromHref(getClusterPath(clusterTabsIds.storage))
+                                .pathname
+                        }
+                    >
+                        <StorageWrapper
+                            parentContainer={container.current}
+                            additionalNodesProps={additionalNodesProps}
+                        />
+                    </Route>
+                    <Route
+                        path={
+                            getLocationObjectFromHref(getClusterPath(clusterTabsIds.versions))
+                                .pathname
+                        }
+                    >
+                        <Versions versionToColor={versionToColor} />
+                    </Route>
+                    <Redirect to={getLocationObjectFromHref(getClusterPath(defaultTab))} />
+                </Switch>
+            </div>
         </div>
     );
 }
 
+function useClusterTab() {
+    const dispatch = useDispatch();
+
+    const defaultTab = useTypedSelector((state) => state.cluster.defaultClusterTab);
+
+    const match = useRouteMatch<{activeTab: string}>(routes.cluster);
+
+    let {activeTab = defaultTab} = match?.params || {};
+    if (!isClusterTab(activeTab)) {
+        activeTab = defaultTab;
+    }
+
+    useEffect(() => {
+        if (activeTab !== defaultTab) {
+            dispatch(updateDefaultClusterTab(activeTab));
+        }
+    }, [activeTab, defaultTab, dispatch]);
+
+    return {activeTab, defaultTab};
```

Comment:
It looks like `activeTab` will be equal to `defaultTab`, since `defaultTab` is updated, if it differs

Reply:
Yes, we update the default tab to the active tab because we want the next time we go to the cluster page to see that tab.

Reply:
But why we need to return both `activeTab` and `defaultTab` from this hook it they are equal?